src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -2858,10 +2858,11 @@ static WellKnownMembers()
                 (byte)(MemberFlags.Method | MemberFlags.Static),                                                                                    // Flags
                 (byte)WellKnownType.ExtSentinel, (byte)(WellKnownType.Microsoft_CodeAnalysis_Runtime_Instrumentation - WellKnownType.ExtSentinel),  // DeclaringTypeId
                 0,                                                                                                                                  // Arity
-                    4,                                                                                                                              // Method Signature
+                    5,                                                                                                                              // Method Signature
```

Comment:
@jaredpar I recall that you had some experience with API versioning. Do you have any recommendations for test-impact? 
One question in particular, is whether the instrumentation runtime library should expose a version number (so the compiler can generate the right code depending on what runtime is there)?